File: coding-test/백준/기본/단어의개수.js
// https://www.acmicpc.net/problem/1152

/**
 * @Date 2024.02.14
 */

const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString();
// const input = 'The Curious Case of Benjamin Button';
// const input = '  ';

const trimInput = input.trim();
console.log(trimInput ? trimInput.split(' ').length : 0);

File: coding-test/백준/기본/단어의개수.py
# https://www.acmicpc.net/problem/1152

# 2024.02.14

import sys

input= sys.stdin.readline()
# input = 'The Curious Case of Benjamin Button';
# input = '  ';

trimInput = input.strip();
if trimInput:
    print(len(trimInput.split(' ')))
else:
    print(0)
